### DISCUSSION TITLE
I cannot play with Focusrite Scarlett One - FRUSTATED !!!

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#2716] [Technical Support & Help] [answered]
Title says it all:

I cannot play with Focusrite Scarlett One - FRUSTATED !!!

Can anyone help me???

Thanks

**Comment by ann0see** [comment]
Hi @luciofernandes1972 
That's bad to hear you're frustrated! What's the exact issue? Do you not hear sound? Do others not hear sound? Is the sound bad? Or anything else?
Please give us an exact error  description. 

The first thing I'd check is that all hardware functions properly, the in and output gain are at decent levels, no mute switch is switched on and the driver is installed. 

Edit: have a look at the troubleshooting page: https://jamulus.io/wiki/Client-Troubleshooting if your problem persists

**Reply by dcorson-ticino-com** [reply]
And remember that to hear any sound in Jamulus you must be connected to a server.

**Reply by HugoVogel** [reply]
Same problem gor me with 2i2 focusrite scarlet

**Reply by ann0see** [reply]
Could you please specify what the problem is exactly?

**Comment by ghost** [comment]
Yes.

The output is only on one side.

Thanks,

Von meinem iPad gesendet

**Reply by ann0see** [reply]
Ok. 
1. Is the pan setting in advanced settings correctly in the middle?
2. Did you set Jamulus‘s "Audiokanäle" to mono in/stereo out or mono? What happens then?
3. Do your headphones work correctly? Test via e.g a YouTube Video
4. Can you set panning on your Interface? Maybe something is wrong there

**Comment by HugoVogel** [comment]
Hi AnnOsee

The problem is.
without logic i try my RFO SCARLETT2-2I2 2 In / 2 Out to connect with Jamulus.
I can receive signal from the by my own created server on Jamulus, but
my send signal f.i. a mike directly plugged in into input of RFO SCARLETT2-2I2 2 In / 2 Out doesn’t connect with Jamulus as discribed in several video’s on youtube.
So something must be wrong in the usb connection with my I mac, because when i use the head phone i hear loud and clear the in and output of de RFO SCARLETT2-2I2 2 In / 2 Out box

thanks for asking me


***@***.***

**Reply by jujudusud** [reply]
Could you please tell us Which OS you are using?
Is it macOS? Which version of macOS? (You tell us it is an I Mac).

**Comment by HugoVogel** [comment]
Macos Mojave


Hugo Vogel
***@***.***

**Reply by jujudusud** [reply]
Do you install the legacy version of Jamulus 3.9.0?

**Comment by HugoVogel** [comment]
yes, 

i try also Jamulus without soundcard on my Macb air. Only de in buit microfoon works on the input of jamulus.
With the soundcard no sound from me reaches Jamulus.

I cannot understand this

Hugo Vogel
***@***.***

**Reply by ann0see** [reply]
Can you take a photo of the interface (including microphone connection) and the audio settings in Jamulus?

**Comment by ann0see** [comment]
Also @HugoVogel and @luciofernandes1972 do you both have the exact same issue?

**Comment by HugoVogel** [comment]
Hugo Vogel
***@***.***

**Reply by ann0see** [reply]
If you uploaded pictures we can't see them. Please log in to GitHub and do not write it as E-Mail since E-Mail only supports text.

**Reply by HugoVogel** [reply]
![Uploading 41F3B8B3-0AA8-4C2A-A20B-4F3674D4E244.jpeg…]()

**Reply by ann0see** [reply]
I can’t see the picture. Maybe you clicked the comment button to quickly. Could you please just drag and drop the image wait 10 seconds and then as soon as there’s no "uploading" anymore click on comment?

**Reply by HugoVogel** [reply]
Another try
![D005F58B-943C-42B7-9546-A19F6F90EFD4](https://user-images.githubusercontent.com/61155911/182979268-fb96226d-b925-43d7-b18a-cebb4a874322.jpeg)
![06CD9812-9B5B-415E-81C1-6DCEC36F7F8F](https://user-images.githubusercontent.com/61155911/182979335-3812a9c4-1c88-470c-aab8-b345349467e5.jpeg)

**Reply by ann0see** [reply]
Ok. Set Audiokanaler to mono in/stereo out, not stereo

**Comment by ann0see** [comment]
@luciofernandes1972 @HugoVogel If you prefer German or French over English, we can also try communicating in another language if that's easier or use an online translator like https://www.deepl.com/

**Reply by HugoVogel** [reply]
I send you the pics 
![2EA16784-5278-4F80-BA7E-82C1D66B9BF0](https://user-images.githubusercontent.com/61155911/183018393-b7fba6f3-212b-443f-a6ba-d6b14bb822b9.png)

**Comment by HugoVogel** [comment]
#
![Uploading FFE1CF61-A266-4ECC-9D3D-B9176BA633FF.jpeg…]()

**Comment by HugoVogel** [comment]
The focusrite has no mono switch


Hugo Vogel
***@***.***
+31646080356

**Reply by dcorson-ticino-com** [reply]
Not on the Interface, in the Jamulus audio setup window.  "Audiokanaler" -> mono-in/stereo out

**Comment by HugoVogel** [comment]
still not working
Hugo Vogel
***@***.***

**Comment by ann0see** [comment]
@HugoVogel Did you try this list: https://github.com/jamulussoftware/jamulus/discussions/2716#discussioncomment-3319857

**Comment by ann0see** [comment]
What happens if you set the switch under the input to "line" instead of "inst"

**Comment by jujudusud** [comment, marked as answer]
https://jamulus.io/wiki/Client-Troubleshooting#using-a-mac-and-your-input-is-not-heard

**Comment by HugoVogel** [comment]
yes
Hugo Vogel
***@***.***

**Reply by ann0see** [reply]
Ok. Did you try to switch the button right below the input to line not instrument?

**Comment by HugoVogel** [comment]
yes i think its my IMac
Hugo Vogel
***@***.***

**Comment by HugoVogel** [comment]
Hi AnnoSee


I have to click on the jamulus privacy button in the preferences of my Imac. So far so good

thanks 

HUGO

**Reply by ann0see** [reply]
Oh. Ok. Good to know. I'll mark @jujudusud 's comment as answer then.

**Reply by HugoVogel** [reply]
But......How can i play a playback file via logic and send it to jamulus only icw focusrite 2i2 and then have it sing in remotely and record it

**Reply by ann0see** [reply]
Use Blackhole. Maybe this document helps you: https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/mobilebasic

**Comment by jujudusud** [comment]
What about @luciofernandes1972 's problem?
Is there an answer too?

**Comment by ghost** [comment]
Hi:

I am really frustrated... and still am.

Currently I am travelling, but this is what I will try: maybe if I feed from the output of the interface to the input of the voice input, then may be I have 2 outputs working via USB and it goes to Jamulus.

Meaning: Guitar input, then feed the output of one of the right or left outputs to the voice input... Then I have 2 inputs (Right and Left). Hope this works.

This really drives me crazy. I just love Jamulus and I stay there just enjoying the other jamming and playing together.

I think Jamulus should get an Focusrite Solo interface to test and to fix this issue.

Best,

Lucio


________________________________
Von: Julien Taverna ***@***.***>
Gesendet: Mittwoch, 10. August 2022 07:55
An: jamulussoftware/jamulus ***@***.***>
Cc: luciofernandes1972 ***@***.***>; Mention ***@***.***>
Betreff: Re: [jamulussoftware/jamulus] I cannot play with Focusrite Scarlett One - FRUSTATED !!! (Discussion #2716)


What about @luciofernandes1972<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fluciofernandes1972&data=05%7C01%7C%7Ca719f11e9e04449c680108da7aa5a3ab%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637957149069030116%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FHYTicNiVTgHgyujXAAlG%2BhaYiY78%2FFiDrzXbEGQiWM%3D&reserved=0> 's problem?
Is there an answer too?

**Reply by jujudusud** [reply]
You have said, before your last post:
"I cannot play with Focusrite Scarlett One.
The output is only on one side."

But you didn't answer to the questions or try what is listed by ann0see or me. Please try:

1. Is the pan setting in advanced settings correctly in the middle?
2. Did you set Jamulus‘s "Audiokanäle" to mono in/stereo out or mono? What happens then?
3. Could you please tell us Which OS you are using? If windows, which version? if macOS, which version? if Linux which distro and version?

Please come to github to answer on the same thread as it is easier to follow.

**Comment by HugoVogel** [comment]
Thanks


Hugo Vogel
***@***.***

**Comment by HugoVogel** [comment]
nothing
Hugo Vogel
***@***.***

**Comment by HugoVogel** [comment]
﻿Yes i did. 
On my iphone it works. 
It seems that jamulus has only a mike level input 


Hugo Vogel
***@***.***
+31646080356





Hugo Vogel
***@***.***
+31646080356

**Reply by dcorson-ticino-com** [reply]
Jamulus does not know what kind of input level you are using, it's input is digital and it is the sound interface that does the translation.  Something in your Scarlet is messed up, but we know that the Scarlet works with Jamulus as many other people are using them.
I hope that someone using a Scarlet can help more.
